Demo/Demo.xcodeproj/project.pbxproj

```diff
@@ -243,7 +243,7 @@
 			isa = PBXCopyFilesBuildPhase;
 			buildActionMask = 2147483647;
 			dstPath = "";
-			dstSubfolderSpec = 16;
+			dstSubfolderSpec = 10;
```

Comment:
mmm.. why not 11?

Reply:
11? That is just crazy! 🙃😂 

I was thinking of 8 at first, but 10 worked much better! 😜